### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Comment by rex4539** [comment]
@gregnazario Could you please take a look?

File: docs/@aptos-labs/ts-sdk-0.0.8/classes/TransactionWorker.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
 </div></section><section class="tsd-index-section"><h3 class="tsd-index-heading">Properties</h3><div class="tsd-index-list"><a href="TransactionWorker.html#account" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="#icon-1024"></use></svg><span>account</span></a>
-<a href="TransactionWorker.html#accountSequnceNumber" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="#icon-1024"></use></svg><span>account<wbr/>Sequnce<wbr/>Number</span></a>
+<a href="TransactionWorker.html#accountSequenceNumber" class="tsd-index-link"><svg class="tsd-kind-icon" viewBox="0 0 24 24"><use href="#icon-1024"></use></svg><span>account<wbr/>Sequence<wbr/>Number</span></a>
```
Comment (gregnazario) [inline docs/@aptos-labs/ts-sdk-0.0.8/classes/TransactionWorker.html:3]: Please don't change this in the documentation.

Unfortunately, looks like the name is misspelled on the actual TransactionWorker, and that will need to be updated in the next release.  At that point the docs will be regenerated correctly.

My suggestion here would be to leave all the existing docs alone, fix any other spelling mistakes, and then with the next release, we'll update the autogenerated documentation for the next release.

**Review by gregnazario — COMMENTED** [review]
Looks good, other than we shouldn't change the generated documentation.  cc @0xmaayan

File: tests/e2e/api/digitalAsset.test.ts [hunk 2]
```diff
@@ -128,31 +128,31 @@ describe("DigitalAsset", () => {
   });
 
   test("it freezes transfer ability", async () => {
-    const transaction = await aptos.freezeDigitalAssetTransaferTransaction({
+    const transaction = await aptos.freezeDigitalAssetTransferTransaction({
```
Comment (0xmaayan) [inline tests/e2e/api/digitalAsset.test.ts:131]: need to also change it on the api file https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/api/digitalAsset.ts#L328-L345

And should also marked as a breaking change on CHANGELOG

**Review by 0xmaayan — COMMENTED** [review]
Overall looks good, thanks for working on it!

There is a function API rename that needs to be renamed on the API file as well and marked as a breaking change on CHANGELOG.
Also, please revert docs changes as it is auto generated.

**Comment by gregnazario** [comment]
> Thanks for the review!
> 
> Phew, it's a lot of hassle to revert the docs changes.
> 
> I better close it :)

Oh, it should have been as simple as `git checkout origin/main docs`. I think

**Pull request closed without merging**